arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -230,7 +232,7 @@ public class NonEmptyList<out A>(
         "import arrow.core.toOption"
       )
     )
-    @JvmStatic
+    @JvmStatic @JvmName("fromList")
     public fun <A> fromList(l: List<A>): Option<NonEmptyList<A>> =
       if (l.isEmpty()) None else Some(NonEmptyList(l))
```

Comment:
I thought these constructors were also deprecated in favour of `toEmptyListOrNone()` and `arrow.core.toNonEmptyListOrNull`.